data/saas/config/domo_config.yml

```diff
@@ -0,0 +1,75 @@
+saas_config:
+  fides_key: domo_connector_example
+  name: Domo SaaS Config
+  description: A sample schema representing the Domo connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: client_id
+    - name: client_secret
+    - name: email
```

Comment:
Remove email identity, same reasoning as the Gorgias connector https://github.com/ethyca/fidesops/pull/304/files#r830290394